### ISSUE TITLE
Cypress eventRegistration test fails (canvas width)

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2271]
## Where to find the issue

Cypress test suite
[eventRegistration.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/pages/eventRegistration.js).

## Describe the issue

The Cypress test [eventRegistration.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/pages/eventRegistration.js) fails with the error:

`expected <canvas#eventqrcode.d-none.eventqr-preview> to have attribute width with the value '827', but the value was '1654'`

A related issue https://github.com/corona-warn-app/cwa-website/issues/2139 was already reported and resolved.

## Steps to reproduce

`npm run test:open`

In Cypress console select and run:

`eventRegistration.js`

## Suggested change

The Cypress test [eventRegistration.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/pages/eventRegistration.js) should be reviewed and updated to work with the current [Event Registration form](https://www.coronawarn.app/en/eventregistration/).

It looks like the changes from PR #2105 "Reintroducing multiQR functionality" broke the Cypress test.

Consider also making sure that the test covers the [single](https://www.coronawarn.app/en/eventregistration/#single) and the [multi QR code](https://www.coronawarn.app/en/eventregistration/#multi) tabs of the [Event Registration form](https://www.coronawarn.app/en/eventregistration/).

**Comment by MikeMcC399** [comment]
Happy New Year to everybody!

I'm not planning to submit a PR for this. I did try to just change the width, but then another error came up:

```
#printCode
expected <button#printCode.btn.btn-secondary> not to have attribute disabled
```

It would be better for somebody who is familiar with the form and with Cypress tests to propose any changes.

**Comment by brifemu** [comment]
@MikeMcC399 

Happy new year! I'm checking this and discovered the origin of

```
#printCode
expected <button#printCode.btn.btn-secondary> not to have attribute disabled
```

This because of PR #2260, the start date inserted by the test is invalid due to new date check. I'm working on it, will be a new PR soon.

**Comment by dsarkar** [comment]
@MikeMcC399 @brifemu Happy new year. Thanks for report, analysis, and solution!

**Comment by MikeMcC399** [comment]
@brifemu 
Thank you very much for picking this up! I wish you and @dsarkar a good start in the New Year!

**Comment by MikeMcC399** [comment]
PR #2277 resolved this issue.

**Comment by MikeMcC399** [comment]
- PR #2730 has caused this issue to re-appear
- PR #2740 is submitted to resolve the test issue

**Comment by MikeMcC399** [comment]
- The issue is now resolved through PR #2740.
